2011/counties/20111108__ms__general__webster__precinct.csv

```diff
@@ -370,26 +370,207 @@ Webster,Fame,State House,35,Libertarian,"Jan R. 'Jay' Butler, Jr.",19
 Webster,Clarkson,State House,35,Libertarian,"Jan R. 'Jay' Butler, Jr.",19
 Webster,Cumberland,State House,35,Libertarian,"Jan R. 'Jay' Butler, Jr.",20
 Webster,Mantee,State House,35,Libertarian,"Jan R. 'Jay' Butler, Jr.",10
-Webster,,Governor,,Republican,Phil Bryant,3459
-Webster,,Governor,,Democrat,Johnny L. DuPree,946
-Webster,,Lieutenant Governor,,Republican,Tate Reeves,3760
-Webster,,Lieutenant Governor,,Reform,Tracella Lou O'Hara Hill,408
-Webster,,Secretary of State,,Republican,Delbert Hosemann,3916
-Webster,,Attorney General,,Democrat,Jim Hood,2564
-Webster,,Attorney General,,Republican,Steve Simpson,1810
-Webster,,State Auditor,,Republican,Stacey E. Pickering,3489
-Webster,,State Auditor,,Reform,Ashley Norwood,673
-Webster,,Treasurer,,Republican,Lynn Fitch,3157
-Webster,,Treasurer,,Democrat,Connie Moran,1025
-Webster,,Treasurer,,Reform,Shawn O'Hara,133
-Webster,,Commissioner of Agriculture & Commerce,,Democrat,Joel Gill,1165
-Webster,,Commissioner of Agriculture & Commerce,,Republican,Cindy Hyde-Smith,3063
-Webster,,Commissioner of Agriculture & Commerce,,Reform,Cathy L. Toole,95
-Webster,,Commissioner of Insurance,,Republican,Mike Chaney,3272
-Webster,,Commissioner of Insurance,,Democrat,Louis Fondren,891
-Webster,,Commissioner of Insurance,,Reform,Barbara Dale Washer,161
-Webster,,State Senate,15,Republican,Gary Jackson,3819
-Webster,,State House,23,Republican,Charles Jim Beckett,698
-Webster,,State House,23,Democrat,Steve Whitten,443
-Webster,,State House,35,Republican,Joey Hood,2727
-Webster,,State House,35,Libertarian,"Jan R. 'Jay' Butler, Jr.",415
```

Comment:
Was there a reason for the above entries to be removed?

Reply:
Yes, those rows are total results (no precinct).